### PR TITLE
Avoid stackdumps in glooctl check

### PR DESCRIPTION
**Pull request by MitchAman**
# Description

Prevent glooctl check from producing stackdumps by removing usages of `Must`.

# Checklist:

- [x] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/master/changelogutils) which references the issue that is resolved.
- [x] If I updated APIs (our protos) or helm values, I ran `make -B install-go-tools generated-code` to ensure there will be no code diff
- [x] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- [x] I opened a draft PR or added the work in progress label if my PR is not ready for review
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works

BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/5061

### CODE REVIEW

File: changelog/v1.11.0-beta11/glooctl-check-fix.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+changelog:
+  - type: NON_USER_FACING
```
Comment (jenshu) [inline changelog/v1.11.0-beta11/glooctl-check-fix.yaml:2]: i would say this is a FIX

Comment (MitchAman) [reply changelog/v1.11.0-beta11/glooctl-check-fix.yaml]: Agreed - meant to change that after copy/pasting a different changelog. Thanks!

**Review by sam-heilbron — APPROVED** [review]

**Comment by cdancy** [comment]
Is there an ETA on when this will make it into a release?

**Pull request merged**